biohansel/subtype/subtyper.py

```diff
@@ -166,7 +167,7 @@ def parallel_query_contigs(input_genomes: List[Tuple[str, str]],
 
```

Comment:
when you have df['scheme'], which value actually gets used? scheme_name or scheme, when both are defined?

Reply:
Also, perhaps better variable names for refpositions and subtypes on line 138?

Reply:
`scheme_name` is used if both are defined. The order in the `<var1> or <var2>`  dictates which is used preferentially. 

Better var names for `refpositions` and `subtypes`? Do you mean the `x` and `y` being used in the list comprehensions?

Reply:
ok yeah that makes sense, yeah for 'x' and 'y'

Reply:
I usually go with short var names for list and dict comprehensions since there's no chance that they can be accessed outside of the comprehension. I was hoping it was clear that the result from  de-structuring `Series.str.split` produces 2 elements and that what `x` and `y` are is clear from the name of the vars they end up in.  The names can be changed if this is potentially confusing. It may be a good idea to extract this code into a separate function with support for custom scheme tile naming conventions since right now we're expecting a very specific format.

Reply:
oh alright yeah that makes sense, I'm good either way, but I agree that putting it in another separate function would be useful